src/Advection/vector_invariant_cross_upwinding.jl

```diff
@@ -0,0 +1,39 @@
+const VectorInvariantCrossVerticalUpwinding = VectorInvariant{<:Any, <:Any, <:Any, <:Any, <:AbstractUpwindBiasedAdvectionScheme, <:CrossAndSelfUpwinding}
+
+#####
+##### Cross upwinding results in the largest kinetic energy content,
+##### but because of presence of mixed upwinding leading to cross-double derivatives
+##### it is slightly unstable at larger orders. 
```

Comment:
```suggestion
##### it is unstable at larger orders. 
```

How is it possible to be "slightly" unstable? Stability is a binary concept, isn't it?